front/app/containers/ProjectsShowPage/shared/survey/TypeformSurvey.tsx

```diff
@@ -45,37 +30,16 @@ interface Props {
 
 const TypeformSurvey = memo<Props>(
   ({ typeformUrl, email, user_id, className }) => {
-    const [isIframeLoaded, setIsIframeLoaded] = useState(false);
-    const windowSize = useWindowSize();
-    const smallerThanLargeTablet = windowSize
-      ? windowSize.windowWidth <= viewportWidths.largeTablet
-      : false;
-
+    const isLargeTablet = useBreakpoint('largeTablet');
     const queryString = stringify(omitBy({ email, user_id }, isNil));
-    const surveyUrl = `${typeformUrl}?${queryString}`;
-
-    const handleIframeOnLoad = () => {
-      setTimeout(() => {
-        setIsIframeLoaded(true);
-      }, 1000);
-    };
+    const surveyUrl = `${typeformUrl}?${queryString}&disable-auto-focus=true`;
```

Comment:
This prevents the iframe from stealing focus which was the reason to add `display: none` to it initially. This in turn was breaking the forms on mobile. Should be good now :)